lib/hooks/create/project.js

```diff
@@ -94,6 +117,7 @@ module.exports = settings => {
       case 'grant':
       case 'transfer':
         return Promise.resolve()
+          .then(() => markContinuation(model))
```

Comment:
Does this run on resubmission as well creation?

It will need to to ensure that any changes in the version are reflected on the task.

Reply:
yes, resubmissions too